09LessonNine/in-class-project/service.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Landscaping and More</title>
+  <link rel="stylesheet" href="css/css-reset.css">
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body class="service">
+  <main class="clearfix">
+<article>
+<div class="textbox two">
+  <h1>GROWING</h1>
+  <h2>We Do It Cause It Is Our Passion</h2>
```

Comment:
The only thing I can see to change on your second page is getting the first h2 in the top left box to be left justified with its h1. Maybe give it its own class and add extra margin left or padding left. Great job!